src/utils.js

```diff
@@ -0,0 +1,9 @@
+const generateId = (array) => {
+  if (array.length) {
```

Comment:
This verification works, but it's not good practice. When comparing a `0`, you should never "abuse" the fact that it's a falsy. If you're dealing with numbers, use `array.length > 0`